dev/benchmarks/complex_layout/android/app/build.gradle

```diff
@@ -19,15 +19,11 @@ apply plugin: 'com.android.application'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion 29
-
-    lintOptions {
-        disable 'InvalidPackage'
-    }
+    compileSdkVersion 30
```

Comment:
this is ok but just for my understanding, this is orthogonal to a agp move right?

Reply:
that's true